### PR TITLE
Update the isRemoteXact flag into a independent bool

### PR DESCRIPTION
**Pull request by poojanilangekar**

### CODE REVIEW

File: src/backend/storage/lmgr/proc.c [hunk 1]
```diff
@@ -442,6 +442,7 @@ InitProcess(void)
 	Assert(pg_atomic_read_u32(&MyProc->clogGroupNext) == INVALID_PGPROCNO);
 
 	pg_atomic_init_u64(&MyProc->assignedXidCsn, InProgressXidCSN);
```
Comment (ctring) [inline src/backend/storage/lmgr/proc.c:444]: Could you add `/* Remotexact */` above this line so we can find it easier later?

File: src/include/storage/proc.h [hunk 2]
```diff
@@ -278,6 +277,18 @@ struct PGPROC
 	 */
 	CSN_atomic assignedXidCsn;
 
+	/*
+	 * A boolean flag to indicate whether or not the xact run on this process
```
Comment (ctring) [inline src/include/storage/proc.h:281]: Add the tag here too.

**Review by ctring — APPROVED** [review]

File: src/include/storage/proc.h (hunk 2 above)
Comment (poojanilangekar) [reply src/include/storage/proc.h:282]: Done.

File: src/backend/storage/lmgr/proc.c (hunk 1 above)
Comment (poojanilangekar) [reply src/backend/storage/lmgr/proc.c:444]: Done.

**Pull request merged**